app/assets/javascripts/app/form-validation.js

```diff
@@ -11,6 +11,15 @@ document.addEventListener('DOMContentLoaded', () => {
 
   if (forms.length !== 0) {
     [].forEach.call(forms, function(form) {
+      form.addEventListener('submit', function() {
+        const buttons = form.querySelectorAll('input[type="submit"]');
```

Comment:
Because a form can only have one submit button, I think you can boil this down:
`form.querySelector('[type=submit]').setAttribute('disabled', 'disabled')`
this will also work for both button and input, but we WILL to remember if we use button to supply `type` rather than leave it implicit.

Reply:
A form can have two submit buttons: https://www.w3schools.com/tags/att_button_formaction.asp

Reply:
While technically you are correct, you're not going to see any such examples in the wild. I'm not even sure the formaction attribute is fully supported. It may be, but then, I've yet to see an example in the wild. I'd vote for the more efficient selector rather than iterating through objects that won't be there. But since you are correct, and we _could_ do such a thing, if you want to keep it as-is, I won't fuss. 

However, I do recall now that the UX guidance was to provide some sort of "working" messaging to help the user know what's going on. I'm not sure where exactly we landed on that, but I do recall it being requested.

Reply:
w.r.t. verbiage I responded to the issue https://github.com/18F/identity-private/issues/2457  and thought that was a bad idea.  While it's great for ajax submits it's bad for non-ajax submits (which is what we have).  This is because if the user hits the back button after a non-ajax submit they will see our verbiage ie "please wait" and wind up waiting indefinitely.  There is no way for the page to respond from a back button and get rid of the message.  I recommended that we could offer some kind of small "reset" type button that just refreshes the page if the user somehow hits back button and sees a disabled button. However, most forms offer links that will get them back to where they were. Not ideal but it works.

Reply:
If they're seeing the text, the button is disabled, right? If we don't put the text in, we still have the disabled button problem, don't we?

Do we need to iterate through forms and enable onload to capture these sorts of errors?

These things are never as simple as I want them to be :-|